src/main/java/meteordevelopment/meteorclient/utils/Utils.java

```diff
@@ -72,7 +72,6 @@ public class Utils {
     public static double frameTime;
     public static Screen screenToOpen;
     public static VertexSorter vertexSorter;
-
```

Comment:
Why. If you want to cleanup then move the 2 constants to the top of the file and with a new line to separate constants and static variables.